src/vs/workbench/contrib/chat/browser/chatContentParts/toolInvocationParts/chatTerminalToolProgressPart.ts

```diff
@@ -688,40 +651,38 @@ class ChatTerminalToolOutputSection extends Disposable {
 		return this._container.classList.contains('expanded');
 	}
 
-	private readonly _container: HTMLElement;
-	private readonly _title: HTMLElement;
-	private readonly _displayCommand: string;
-	private readonly _terminalData: IChatTerminalToolInvocationData;
-	private readonly _accessibleViewService: IAccessibleViewService;
 	private readonly _onDidChangeHeight: () => void;
 	private readonly _ensureTerminalInstance: () => Promise<ITerminalInstance | undefined>;
 	private readonly _resolveCommand: () => ITerminalCommand | undefined;
-	private readonly _getTerminalTheme: () => { background?: string; foreground?: string } | undefined;
-	private readonly _getStoredCommandId: () => string | undefined;
 
 	private readonly _outputBody: HTMLElement;
 	private _outputScrollbar: DomScrollableElement | undefined;
-	private _outputContent: HTMLElement | undefined;
 	private _outputResizeObserver: ResizeObserver | undefined;
 	private _renderedOutputHeight: number | undefined;
-	private _lastOutputTruncated = false;
 	private readonly _outputAriaLabelBase: string;
+	private _mirror: DetachedTerminalCommandMirror | undefined;
+	private _contentContainer: HTMLElement | undefined;
+	private _terminalHost: HTMLElement | undefined;
+	private _emptyElement: HTMLElement | undefined;
+	private _hasRendered = false;
 
 	private readonly _onDidFocusEmitter = new Emitter<void>();
 	private readonly _onDidBlurEmitter = new Emitter<FocusEvent>();
 
-	constructor(options: ChatTerminalToolOutputSectionOptions) {
+	constructor(
+		private readonly _container: HTMLElement,
```

Comment:
Exposing `domNode` on `ChatTerminalToolOutputSection` would align closer to how `ChatTerminalToolProgressPart` works and avoid giving `ChatTerminalToolOutputSection` a pointer to an element it doesn't own.